frontend/src/components/ModifyMotoForm.vue

```diff
@@ -123,7 +142,20 @@ export default {
         })
         .catch((error) => {
           console.error(error)
-          alert('Error modifying: please check that license plate is not equal to another moto or that the moto exists')
+          switch (error.status) {
+            case 409:
+              alert('Please check that license plate is not equal to another moto')
+              break
+            case 400:
+              alert('Please check that State and Battery fields has sense')
```

Comment:
Puedes poner: "Please check that the state and the battery fields are consistent"? @lwangwan 
Aunque este comportamiento te lo he comentado en la API